packages/block-template/templates/custom-element/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "allowSyntheticDefaultImports": true,
     "experimentalDecorators": true,
     "isolatedModules": true,
-    "jsx": "preserve",
+    "jsx": "react-jsx",
```

Comment:
As above RE typescript/JSX.

Reply:
Replied in https://github.com/blockprotocol/blockprotocol/pull/441#r925489980